typings/src/ui/ui-events.ts

```diff
@@ -42,6 +42,7 @@ export type EventDataTypes = {
   name: string;
   version?: string;
   entryId?: string;
+  commentId?: string;
```

Comment:
Can you use the entryId here instead of creating a new key? This is the way it was suppose to work (yeah, I think the naming is misleading). We'll need to rename the entryId to itemId so it should reflect that it's generic. If it's not much you can do this change (or leave it for future but maybe add a comment there instructing to use the entryId)

Reply:
For the inline editor, sometimes we need both the post id and comment id and entryId is scalar in this case. There is generic type in the `EventDataTypes` but I needed the type check when I am mounting the inline editor using `Extension` component. May be we do `entryId?: string | Record<"postId" | "commentId", string>` as a type ?

Reply:
Having read your comment below, I will take out the commentId and leave only entryId as it is.